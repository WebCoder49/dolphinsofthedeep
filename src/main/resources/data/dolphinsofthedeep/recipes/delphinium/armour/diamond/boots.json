{
    "type": "minecraft:crafting_shaped",
    "pattern": [
      "   ",
      "I I",
      "I I"
    ],
    "key": {
      "I": {
        "item": "dolphinsofthedeep:diamond_delphinium_ingot"
      }
    },
    "result": {
      "item": "dolphinsofthedeep:diamond_delphinium_boots",
      "count": 1
    }
  }