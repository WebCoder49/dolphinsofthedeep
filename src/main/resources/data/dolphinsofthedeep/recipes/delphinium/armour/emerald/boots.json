{
    "type": "minecraft:crafting_shaped",
    "pattern": [
      "   ",
      "I I",
      "I I"
    ],
    "key": {
      "I": {
        "item": "dolphinsofthedeep:emerald_delphinium_ingot"
      }
    },
    "result": {
      "item": "dolphinsofthedeep:emerald_delphinium_boots",
      "count": 1
    }
  }