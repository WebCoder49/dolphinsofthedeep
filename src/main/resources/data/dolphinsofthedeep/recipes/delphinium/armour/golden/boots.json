{
    "type": "minecraft:crafting_shaped",
    "pattern": [
      "   ",
      "I I",
      "I I"
    ],
    "key": {
      "I": {
        "item": "dolphinsofthedeep:golden_delphinium_ingot"
      }
    },
    "result": {
      "item": "dolphinsofthedeep:golden_delphinium_boots",
      "count": 1
    }
  }